mergify_engine/context.py

```diff
@@ -745,31 +745,58 @@ async def _get_consolidated_data(self, name):
             return [r["user"]["login"] for r in comments if r["state"] == "COMMENTED"]
 
         # NOTE(jd) The Check API set conclusion to None for pending.
-        # NOTE(sileht): "pending" statuses are not really trackable, we
-        # voluntary drop this event because CIs just sent they status every
-        # minutes until the CI pass (at least Travis and Circle CI does
-        # that). This was causing a big load on Mergify for nothing useful
-        # tracked, and on big projects it can reach the rate limit very
-        # quickly.
```

Comment:
The rate limit and big load issue are not true anymore. We already process pending statuses since a while now (even if the attribute is not there yet).